install-script.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+PWD=$(pwd)
+
+# first, backup any existing scripts if they are not symlinks
```

Comment:
@jordwalke I'm guessing that person is probably a little more sophisticated (both technically and with their vim setup) than the average person who would be using this script. If someone already has a symlink setup, I would be more comfortable telling them to manually take care of their backups, because if it's symlinked, it's already backed up "remotely" anyway.